packages/react-native-windows-init/src/Cli.ts

```diff
@@ -0,0 +1,291 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @format
+ */
+
+import * as yargs from 'yargs';
+import * as fs from 'fs';
+import * as semver from 'semver';
+import {execSync} from 'child_process';
+import * as validUrl from 'valid-url';
+import * as prompts from 'prompts';
+import * as findUp from 'find-up';
+import * as chalk from 'chalk';
+
+const Registry = require('npm-registry');
```

Comment:
nit: if we're doing ES6 imports already, can we do `import * as Registry from 'npn-registry'`? Seems like we're missing types for this (unless they're in package), but if this was meant to suppress that we could always just add an explicit suppression.